elastic/kibana
# [Ingest Manager] Index pattern installation use requested package version

# Summary
This PR addresses this issue: https://github.com/elastic/kibana/issues/80022

The index pattern installation process previously requested the latest package version to retrieve the fields for installing the `logs-*` and `metrics-*` kibana index patterns. It ignored the requested package version passed into the function. This PR fixes it to use the passed in version if it was specified.

## Testing
1. Start a fresh kibana and elasticsearch
2. Force install the endpoint package version `0.13.1`

```
curl --location --request POST 'http://elastic:changeme@localhost:5601/api/fleet/epm/packages/endpoint-0.13.1' \
--header 'Accept: application/json' \
--header 'Content-Type: application/json' \
--header 'kbn-xsrf: xxx' \
--data-raw '{
    "force": true
}'
```

3. Navigate to the index patterns in stack management and search for a term that should not exist in that version of the endpoint package (e.g. `caseless` fields were added in `0.16.0`)

![image](https://user-images.githubusercontent.com/56361221/95512626-1e636f00-0987-11eb-975b-2bd8e73edef9.png)
